cicd/pipelines/op-release-notes.adoc

```diff
@@ -19,7 +19,30 @@ toc::[]
 For an overview of {pipelines-title}, see xref:../../cicd/pipelines/understanding-openshift-pipelines.adoc#understanding-openshift-pipelines[Understanding OpenShift Pipelines].
 
 [id="getting-support"]
-== Getting support
+== Pipelines support
+
+This table lists down the various {pipelines-title} components along with the version compatibility matrix.
+
+|===
+|{pipelines-title} | Support status | Pipelines | Triggers | CLI | Operator | Tekton Catalog | {product-title}
```

Comment:
We don't prefix Pipelines, Triggers and CLI with `Tekton`, I would omit it also in Catalog

Reply:
> We don't prefix Pipelines, Triggers and CLI with `Tekton`, I would omit it also in Catalog

@ppitonak Thanks for the comments. Incorporated all the comments. PTAL and kindly provide LGTM.